ngraph/core/reference/include/ngraph/runtime/reference/mvn.hpp

```diff
@@ -73,6 +73,62 @@ namespace ngraph
                            true);
                 }
             }
+
+            template <typename T>
+            void mvn_6(const T* arg,
+                       T* out,
+                       const Shape& in_shape,
+                       AxisSet reduction_axes,
```

Comment:
This function's params CV-qualifiers should be more unified. I think everything except for `out` should become const and additionally the `reduction_axes` should be passed by reference.